Email verification for SSO (#7311)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/invite-users.md

## User experience

:::info Email verification

> Email verification is mandatory for all new users in dbt Cloud, including using Single Sign-On (SSO)⁠⁠. Automatic provisioning without email verification is not allowed. This is a security requirement that cannot be bypassed.

Author: ```suggestion
Email verification is mandatory for all new users in <Constant name="cloud" />, including using Single Sign-On (SSO)⁠⁠. Automatic provisioning without email verification is not allowed. This is a security requirement that cannot be bypassed.
```

:::

> dbt Cloud generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure that traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/access-regions-ip-addresses).

Author: ```suggestion
<Constant name="cloud" /> generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure that traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/access-regions-ip-addresses).
```

## User experience

:::info Email verification

> Email verification is mandatory for all new users in dbt Cloud, this includes using SSO⁠⁠. We don't allow automatic provisioning without email verification. This is a security requirement that cannot be bypassed.

Reviewer: ```suggestion
Email verification is mandatory for all new users in dbt Cloud, including using Single Sign-On (SSO)⁠⁠. Automatic provisioning without email verification is not allowed. This is a security requirement that cannot be bypassed.
```

:::

> dbt Cloud generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/access-regions-ip-addresses).

Reviewer: For the term "dbt Cloud", should we use `<Constant name="cloud" />` instead? 

```suggestion
dbt Cloud generates and sends emails from `support@getdbt.com` to the specified addresses. Make sure that traffic from the `support@getdbt.com` email is allowed in your settings to avoid emails from going to spam or being blocked. This is the originating email address for all [instances worldwide](/docs/cloud/about-cloud/access-regions-ip-addresses).
```

> The email contains a link to create an account. When the user clicks on this they will be brought to one of two screens depending on whether SSO is configured or not.

Reviewer: ```suggestion
The email contains a link to create an account. When the user clicks on this link, they will be brought to one of two screens depending on whether SSO is configured or not.
```

<TabItem value="SSO user">

> If SSO is configured for the environment the user will need to:

Reviewer: ```suggestion
If SSO is configured for the environment, the user must:
```

> 1. Click the link received in their email account.

Reviewer: ```suggestion
1. Click the link in the verification email.
```

> 2. Then, click the option to join the account.

Reviewer: ```suggestion
2. Click the option to join the account.
```

> 3. The user will then be brought to a confirmation screen, and presented with a link to authenticate against the company's identity provider:

Reviewer: ```suggestion
3. A confirmation screen appears, with a link to authenticate against the company's identity provider. Click **Authenticate with your enterprise login**.
```
